### PR TITLE
Checkpoint 4

### PR DESCRIPTION
**Pull request by DarylMcCullough**
Added pages for viewing registered applications, creating them, etc.

### CODE REVIEW

File: test/system/registered_applications_test.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+require "application_system_test_case"
```
Comment (brittanyjspringer) [inline test/system/registered_applications_test.rb]: @DarylMcCullough remove please.

File: app/helpers/registered_applications_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module RegisteredApplicationsHelper
```
Comment (brittanyjspringer) [inline app/helpers/registered_applications_helper.rb]: Remove please.

**Review by brittanyjspringer — CHANGES_REQUESTED** [review]

**Comment by brittanyjspringer** [comment]
@DarylMcCullough can you please post your Test Coverage % as well?

File: test/system/registered_applications_test.rb (hunk 1 above)
Comment (DarylMcCullough) [reply test/system/registered_applications_test.rb]: That was generated automatically. I'm not sure what is the distinction between the `test` directory and the `spec` directory.

**Comment by brittanyjspringer** [comment]
@DarylMcCullough /test is for the Minitest Framework (you are not using this). Spec is for rSpec (you are using this).

**Comment by DarylMcCullough** [comment]
What do you mean by Test Coverage %?

**Comment by brittanyjspringer** [comment]
I thought we had covered using Simplecov in the past but it is possible we haven't.

Please install Simplecov in your test group, add the line to spec_helper.rb and then run your tests. It will generate a % of code covered when you run your tests. 

https://github.com/colszowka/simplecov

**Comment by DarylMcCullough** [comment]
I got Simplecov working. Should I commit the coverage directory so you can see the results?

**Comment by brittanyjspringer** [comment]
@DarylMcCullough no, what you did by adding it to .gitignore is great. I just want you to post this in your comment when you open the PR. 

Example: 

Description: Added pages for viewing registered applications, creating them, etc.
Test Coverage: 92.5%

**Comment by DarylMcCullough** [comment]
[Coverage html](http://dee-mccullough.com/bloc/ez-events/coverage-2017-07-05.jpg)

**Review by brittanyjspringer — APPROVED** [review]

**Pull request merged**